Dockerfile

```
@@ -74,6 +75,10 @@ RUN mv ./istio-${ISTIO_VERSION}/bin/* /usr/local/bin/
 # yq install
 RUN pip3 install yq==${YQ_VERSION}
 
+# jsonnet install
+RUN wget https://github.com/google/jsonnet/releases/download/${JSONNET_VERSION}/jsonnet-bin-${JSONNET_VERSION}-linux.tar.gz 
+RUN tar xzf jsonnet-bin-${JSONNET_VERSION}-linux.tar.gz -C /usr/local/bin/ jsonnet
```

This way, layers would be reduced and archive dowloaded would be removed.
```suggestion
RUN wget https://github.com/google/jsonnet/releases/download/${JSONNET_VERSION}/jsonnet-bin-${JSONNET_VERSION}-linux.tar.gz && \
  tar xzf jsonnet-bin-${JSONNET_VERSION}-linux.tar.gz -C /usr/local/bin/ jsonnet && \
  rm jsonnet-bin-${JSONNET_VERSION}-linux.tar.gz
```